packages/server/lib/browsers/index.ts

```diff
@@ -273,4 +273,15 @@ export = {
 
     return instance
   },
+
+  /**
+   * Closes extra targets that are not the Cypress tab
+   */
+  async closeExtraTargets () {
+    if (!instance || !instance.browser) return
```

Comment:
```suggestion
    if (!instance?.browser) return
```

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/28204/commits/62e08a3addb8246bba0567e5f307c46a0b1a89f9